cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -900,6 +900,20 @@ storage:
       remote:
         url: "data:text/plain;base64,{{ .Cluster.ConfigItems.admission_controller_key }}"
 
+  - filesystem: root
+    path: /etc/kubernetes/ssl/service-account-private-key.pem
+    mode: 0664
+    contents:
+      remote:
+        url: "data:text/plain;base64,{{ .Cluster.ConfigItems.service_account_private_key }}"
+
+  - filesystem: root
+    path: /etc/kubernetes/ssl/service-account-public-key.pem
+    mode: 0664
+    contents:
+      remote:
+        url: "data:text/plain;base64,{{ .Cluster.ConfigItems.service_account_private_key | base64Decode | publicKey | base64 }}"
```

Comment:
My understanding is that this would result in a new key every time CLM is invoked? If so, then we shouldn't do it this way IMO.

Reply:
No, it just writes a public key into a separate file. Not that it's really doing anything for security, but we could always improve it.

Reply:
That is, the output of this is stable so there won't be any random node rotations.

Reply:
> That is, the output of this is stable so there won't be any random node rotations.

Thanks, that is what I meant.